src/posthog-surveys.ts

```diff
@@ -83,76 +86,84 @@ export class PostHogSurveys {
     loadIfEnabled() {
         const surveysGenerator = assignableWindow?.__PosthogExtensions__?.generateSurveys
 
-        if (!this.instance.config.disable_surveys && this._decideServerResponse && !surveysGenerator) {
-            if (this._surveyEventReceiver == null) {
-                this._surveyEventReceiver = new SurveyEventReceiver(this.instance)
-            }
-
+        if (!this.instance.config.disable_surveys && this._surveysEnabledRemotely && !surveysGenerator) {
             assignableWindow.__PosthogExtensions__?.loadExternalDependency?.(this.instance, 'surveys', (err) => {
                 if (err) {
                     return logger.error('Could not load surveys script', err)
                 }
 
+                logger.info('surveys loaded')
+
                 this._surveyManager = assignableWindow.__PosthogExtensions__?.generateSurveys?.(this.instance)
             })
         }
     }
 
-    getSurveys(callback: SurveyCallback, forceReload = false) {
+    private reloadSurveys(callback: SurveyCallback) {
+        logger.info('Reloading surveys')
+
+        // TODO: Move this to RemoteConfig loading instead (or just remove entirely)
+
+        this.instance._send_request({
+            url: this.instance.requestRouter.endpointFor('api', `/api/surveys/?token=${this.instance.config.token}`),
+            method: 'GET',
+            callback: (response) => {
+                if (response.statusCode !== 200 || !response.json) {
+                    return callback([])
+                }
+
+                const surveys = response.json.surveys || []
+
+                this.onSurveys(surveys)
+
+                return callback(surveys)
+            },
+        })
+    }
+
+    private onSurveys(surveys: Survey[]) {
+        this._loadedSurveys = surveys
+        this._surveysEnabledRemotely = surveys.length > 0
```

Comment:
I think `_surveysEnabledRemotely` should be `true` if `surveys` isn't undefined, and empty is a valid state (no surveys but the API call has been made), otherwise it'd be semantically wrong, at least the naming.

Reply:
The thing is I actually think we should just ditch that property altogether. Server side that property is simply "are there surveys to load" but if we now include them in remtoe config I actually think that property is pointless...